src/drive/web/modules/filelist/HammerComponent.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import Hammer from '@egjs/hammerjs'
+import propagating from 'propagating-hammerjs'
+import styles from './fileopener.styl'
+
+class HammerComponent extends Component {
```

Comment:
Can you add a docstring on why we need this component ? It is not clear to me why we have to have Hammer dealing with taps to trigger onClicks ?